src/ui/wxWidgets/I18N/pos/pwsafe_de.po

```diff
@@ -4108,26 +4108,21 @@ msgstr "Das Punktzeichen ist in Gruppennamen nicht erlaubt"
 msgid "Invalid Character"
 msgstr "Ungültiges Zeichen"
 
-#: ../../../ui/wxWidgets/TreeCtrl.cpp:1045
-#: ../../../ui/wxWidgets/TreeCtrl.cpp:1051
-#: ../../../ui/wxWidgets/TreeCtrl.cpp:1057
+#: ../../../ui/wxWidgets/TreeCtrl.cpp:1065
+#: ../../../ui/wxWidgets/TreeCtrl.cpp:1071
 msgid "Drag and Drop failed"
 msgstr "Drag und Drop fehlgeschlagen"
 
-#: ../../../ui/wxWidgets/TreeCtrl.cpp:1045
+#: ../../../ui/wxWidgets/TreeCtrl.cpp:1065
 #, fuzzy
 msgid "Destination is no group"
 msgstr "Zielfestplatte ist voll."
```

Comment:
Would you be willing to update the translation here and then remove the "fuzzy" keyword?

Reply:
I prepared already a change correcting all fuzzy entries (in pwsafe_de.po, but waiting for feedback from wxWidgets. At first I tried by forum, but now opened a ticket.

https://forums.wxwidgets.org/viewtopic.php?p=202975&sid=037f43db4826987975793ecaec1815ca#p202975

https://trac.wxwidgets.org/ticket/19021

There is one left item with the toolbar menu as well. When translated not always the extended menu ">>" is shown, but can be selected. I have to analyze later.

Reply:
The discussion and reported bug are related to a problem with the cursor during drag and drop, if I'm not mistaken. I don't understand how the strings marked as fuzzy relate to this.

Reply:
Text corrected - make pos seam to change to fuzzy wording.